optuna/visualization/matplotlib/_optimization_history.py

```diff
@@ -94,42 +104,71 @@ def _get_optimization_history_plot(
     cmap = plt.get_cmap("tab10")  # Use tab10 colormap for similar outputs to plotly.
 
     # Prepare data for plotting.
-    trials = [t for t in study.trials if t.state == TrialState.COMPLETE]
+    # trials = [t for t in study.trials if t.state == TrialState.COMPLETE]
+    all_trials = list(
```

Comment:
Should we return an empty plot if `studies` is empty by following the Plotly code??